types/post_proof_partitions.go

```diff
@@ -1,8 +1,11 @@
 package types
 
+// PoStProofPartitions TODO(laser)
 type PoStProofPartitions uint64
 
 const (
+	// TestPoStPartitions TODO(laser)
```

Comment:
TestPoStPartitions is an opaque value signaling that an unknown number of partitions were used when creating a PoSt proof in test mode.